### ISSUE TITLE
Bug report

### ISSUE THREAD

**Issue by MattDerby2** [localsend/localsend#2106]
### Describe the bug

Computeractive magazine recommends a file sharing tool called LocalSend to share files between devices on the same wIfi network. When I try to use it devices on the network (iphone and new Dell laptop) cannot discover each other. Guidance suggests that the network "may not allow communication between participants due to Access Point (AP) Isolation. In this case this option must be disabled on the router" I have been told "Sky wireless hardware doesn't do client isolation, so whatever the issue is, it's not that."   So I can't use LocalSend. Any suggestions?

### To reproduce

2. Click on '...'
3. Scroll down to '...'

### Expected behavior

The tool to be usable

### Desktop (please complete the following information)

Windows 11

### Smartphone (please complete the following information)

iphone 13 mini

**Comment by Tienisto** [comment]
Neither Windows nor the iPhone can discover each other?

**Comment by MattDerby2** [comment]
That’s correct. Both devices are on the network but cannot ‘see’ each other

**Comment by JustAdreamerFL** [comment]
@MattDerby2 does discovering work for you on different network?
